AdebayoJesutofunmi.java

```diff
@@ -0,0 +1,19 @@
+package assignment_one;
+
+public class AdebayoJesutofunmi {
+
+	public static void main(String[] args) {
+	System.out.println("Hello everyone.");
+	System.out.println("My name is Adebayo Jesutofunmi from Kwara State,Nigeria");
+	System.out.println("A fun fact about me is I cannot keep my balance when I laugh so my legs give way and I fall down laughing");
+	System.out.println("I love food, and watching anime gives me joy");
+	System.out.println("For a very long time,I have always been fascinated by how computers work and how technology has brought us here today.");
+	System.out.println("That is why I decided to study computing Science in hopes that I would be able to make a geat impact with it one day.");
+	System.out.println();
+	System.out.println("My initials are");
+	System.out.println("JA");

```

Comment:
no ascii art?!?

Reply:
Good evening sir. I had no idea how to do the ASCII art. I didn't also know we were going to talk about it so by the time we did I had already submitted. I'll be doing it again